packages/@o3r/core/package.json

```diff
@@ -127,6 +127,22 @@
     "typescript": "~4.9.4",
     "zone.js": "^0.13.0"
   },
+  "generatorDependencies": {
```

Comment:
Are these packages handled by version-harmonize script ? If not we should add a task to include them.

Reply:
They are handled by both version-harmonized and Renovate

Reply:
nice :)